src/util/cache/getCachedItemPath.ts

```diff
@@ -16,5 +18,16 @@ export default function getCachedItemPath(
   bucket: CacheBucket,
   itemPath: CacheItemPath
 ): string {
-  return join(CACHE_DIR, bucket, ...itemPath);
+  const projectPath = findFileInCurrentPath(EMULSIFY_PROJECT_CONFIG_FILE);
+
+  if (!projectPath) {
+    throw new Error(`Unable to find ${EMULSIFY_PROJECT_CONFIG_FILE}`);
+  }
+
+  return join(
+    CACHE_DIR,
+    bucket,
+    createHash('md5').update(projectPath).digest('hex'),
```

Comment:
This adds a project-specific folder to cached items, so that if you have multiple Emulsify projects using the same system, they'll reference different copies of the system.